credit.py

```diff
@@ -7,16 +7,27 @@
                     format='%(asctime)s %(message)s', 
                     filemode='w') 
 logger = logging.getLogger() 
-logger.setLevel(logging.ERROR) 
+logger.setLevel(logging.WARNING) 
```

Comment:
This code defines global logger settings, it belongs in `main.py`

Reply:
Done.